src/main/antlr4/FIRRTL.g4

```diff
@@ -87,6 +87,16 @@ field
   : 'flip'? id ':' type
   ;
 
+extModuleName
```

Comment:
Yeah I'd rather call it name everywhere but name is already taken by the actual Firrtl name of the ExtModule. I can make it extname everywhere or defname, what do you think?